src/storage/rpc.rs

```diff
@@ -70,6 +70,20 @@ impl<N> RpcStorage<N> {
     pub fn get_cache_block_hashes(&self) -> AHashMap<u64, B256> {
         self.cache_block_hashes.lock().unwrap().clone()
     }
+
+    /// Update the cache of accounts
```

Comment:
Do we really need this? Can the call site just call the get function and apply their changes externally?